server/models/User.js

```diff
@@ -3,6 +3,11 @@ const bcrypt = require("bcrypt");
 const Order = require("./Order");
 
 const userSchema = new Schema({
+    username: {
+        type: String,
+        required: true,
+        trim: true
+    },
```

Comment:
Maybe add a min of 4 chars for username and that it must be unique.

Reply:
done